lib/resque/job_removal.rb

```diff
@@ -0,0 +1,48 @@
+module Resque
+  module JobRemoval
+    # Instruct all workers to kill their current running job
+    # if it matches the given class or class + smart id.
+    #
+    # The smart id can be a string which will be used by a custom
+    # smart id matcher to identify & kill a subset of the class's jobs.
+    def killall(klass, smart_id = nil)
+      op_key = 'kill_ops'
```

Comment:
Maybe this could be a constant? Same for `suspend_ops` below

Reply:
Right! Fixed in the fixup